commands/src/main/java/net/apartium/cocoabeans/commands/parsers/ParserFactory.java

```diff
@@ -17,6 +18,42 @@
 @ApiStatus.AvailableSince("0.0.30")
 public interface ParserFactory {
 
+    /**
+     * Get the class of the parser factory from an annotation
+     * @param annotation The annotation that has a CommandParserFactory
+     * @return The class of the parser factory
+     */
+    @ApiStatus.AvailableSince("0.0.37")
+    static Class<? extends ParserFactory> getParserFactoryClass(Annotation annotation) {
```

Comment:
why do we need this method, it doesn't look like you use it

Reply:
it does